app/sidekiq/education_form/templates/1995.erb

```diff
@@ -184,10 +184,14 @@ Signature/Title/Branch of Armed Forces Education Service Officer     Date
 <% if @applicant.isActiveDuty -%>
 As an active-duty service member, you have consulted with an Education Service
 Officer (ESO) regarding your education program.
+<% else -%>
+<% if @applicant.minorHighSchoolQuestions -%>
+      You are the parent, guardian, or custodian of the applicant
 <% else -%>
       Certification and Signature of Applicant
+<% end -%>
 Signature of Applicant                                               Date
 <% end -%>
 <% end -%>
 
-<%= parse_with_template_path('footer') %>
+<%= parse_with_template_path('footer') %>
```

Comment:
This file is missing a final new line. If you're using VS Code: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files